mlflow/store/tracking/file_store.py

```diff
@@ -798,3 +799,16 @@ def record_logged_model(self, run_id, mlflow_model):
 
     def update_artifacts_location(self, run_id, new_artifacts_location):
         raise ValueError("Artifacts cannot be moved with FileStore backend")
+
+
+def _filter_columns(run, columns_to_whitelist):
+    run_data = run.data
+    if columns_to_whitelist is not None:
```

Comment:
nitpick: what about inverting the logic of this condition so that you reduce the number of lines of nested code? wdyt?
```suggestion
    if columns_to_whitelist is None:
        return run
    ....
```

Reply:
Done